client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -42,13 +42,15 @@ module.exports = React.createClass( {
 	},
 
 	componentDidMount() {
+		if ( ! this.props.selectedSite || ! this.props.selectedSite.jetpack || ! this.props.selectedSite.canManage() ) {
+			return;
```

Comment:
maybe if `! this.props.selectedSite.canManage()` we could render a `jetpack-manage-error-page` explaining what's going on? @rickybanister